langdetect/detector_factory.py

```diff
@@ -116,22 +120,26 @@ def get_lang_list(self):
 
 PROFILES_DIRECTORY = path.join(path.dirname(__file__), 'profiles')
 _factory = None
+_supported_languages = []
```

Comment:
Same comment here and below regarding initializing supported_languages as None instead of empty list